libs/input-mapping/src/Staging/StrategyFactory.php

```diff
@@ -42,6 +43,9 @@ public function __construct(
             StagingType::Abs,
             StagingType::WorkspaceSnowflake,
             StagingType::WorkspaceBigquery => FileLocal::class,
+            StagingType::None => throw new StagingException(
+                'Staging type "none" is not supported.',
+            ),
```

Comment:
?

Reply:
To by mělo byt v cajku, řešil sem to tady s pepů.
https://keboolaglobal.slack.com/archives/C055HSMKX51/p1754489131137279